pages/claim/[type]/[code].tsx

```diff
@@ -94,20 +94,21 @@ function ClaimDonation({}: Props): ReactElement {
       code: code,
     };
     if (contextLoaded && user) {
-      const userLang = localStorage.getItem('language') || 'en';
       postAuthenticatedRequest(
-        `/api/v1.3/${userLang}/validateCode`,
+        `/app/redeem`,
         submitData,
         token,
         handleError
       ).then((res) => {
         if (res.code === 401) {
           setErrorMessage(res.message);
           setIsUploadingData(false);
-        } else if (res.status === 'error') {
-          setErrorMessage(res.errorText || t('me:wentWrong'));
+        } else if (res.error_code === 'already_redeemed') {
+          setErrorMessage(res.error_code || t('me:wentWrong'));
```

Comment:
Earlier, t('me:wentWrong') was used as a catch all message in case no errorText was set.

The error handling logic has changed now, so the way this is used will change as well.